spec/models/users_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do 
+
+  describe 'validations' do 
```

Comment:
for email, this can be changed to:
 validates :email, presence: true, uniqueness: true